backend/tests/slack/commands/management/slack_sync_messages_test.py

```
@@ -0,0 +1,638 @@
+
+        mock_client = Mock()
+        mock_web_client.return_value = mock_client
+        mock_client.conversations_history.return_value = mock_slack_history_response_final
+
+        stdout = StringIO()
+        with (
+            patch.object(Workspace.objects, "all", return_value=mock_workspaces),
+            patch.object(Conversation.objects, "filter", return_value=mock_conversations),
+            patch.object(Message.objects, "filter") as mock_message_filter,
+            patch.object(Message, "update_data") as mock_update_data,
+            patch.object(Message, "bulk_save") as mock_bulk_save,
```

Ruff (0.11.9)

Local variable `mock_bulk_save` is assigned to but never used

Remove assignment to unused variable `mock_bulk_save`

(F841)